alvr/server_openvr/cpp/platform/win32/shared/d3drender.cpp

```diff
@@ -90,7 +90,7 @@ namespace
 	public:
 		CEventHelper()
 		{
-			UuidFromString( ( RPC_CSTR ) "8c8f13b1-60eb-4b6a-a433-de86104115ac", &guid );
+			UuidFromStringA( ( RPC_CSTR ) "8c8f13b1-60eb-4b6a-a433-de86104115ac", &guid );
```

Comment:
I'm pretty sure this should be fine, however I have no real way to verify it. What's the error you get when trying to build?
If the `UNICODE` preprocessor definition doesn't change the char width (fucking utf16), then it should be fine.